packages/flutter/test/services/platform_views_test.dart

```diff
@@ -142,9 +141,87 @@ void main() {
       await viewController.setLayoutDirection(TextDirection.rtl);
       expect(
           viewsController.views,
-          unorderedEquals(<FakePlatformView>[
-            FakePlatformView(0, 'webview', const Size(100.0, 100.0), AndroidViewController.kAndroidLayoutDirectionRtl),
+          unorderedEquals(<FakeAndroidPlatformView>[
+            FakeAndroidPlatformView(0, 'webview', const Size(100.0, 100.0), AndroidViewController.kAndroidLayoutDirectionRtl),
+          ]));
+    });
+  });
+
+  group('iOS', ()
+  {
+    FakeIosPlatformViewsController viewsController;
+    setUp(() {
+      viewsController = FakeIosPlatformViewsController();
+    });
+
+    test('create iOS view of unregistered type', () async {
+      expect(
+            () {
+          return PlatformViewsService.initUiKitView(
+            id: 0,
+            viewType: 'web',
+            layoutDirection: TextDirection.ltr,
+          );
+        },
+        throwsA(isInstanceOf<PlatformException>()),
+      );
+    });
+
+    test('create iOS views', () async {
+      viewsController.registerViewType('webview');
+      await PlatformViewsService.initUiKitView(
+          id: 0, viewType: 'webview', layoutDirection: TextDirection.ltr);
+      await PlatformViewsService.initUiKitView(
+          id: 1, viewType: 'webview', layoutDirection: TextDirection.rtl);
+      expect(
+          viewsController.views,
+          unorderedEquals(<FakeUiKitView>[
+            FakeUiKitView(0, 'webview'),
+            FakeUiKitView(1, 'webview'),
+          ]));
```

Comment:
trailing commas might get the formatter to return something cleaner

Reply:
done